cstool/cstool.h

```diff
@@ -0,0 +1,23 @@
+#ifndef CAPSTONE_CSTOOL_CSTOOL_H_
+#define CAPSTONE_CSTOOL_CSTOOL_H_
+
+void print_insn_detail_x86(csh ud, cs_mode mode, cs_insn *ins);
```

Comment:
Arent these definitions depending on ifdefs for each selected arch? I think those must be wrapped around ifdefs

Reply:
But `cstool` uses `cs_support(CS_ARCH_XXX)` to query capstone for architectural support, not conditional compilation.

Reply:
Hi, can this thread be marked as solved?